src/types.ts

```diff
@@ -68,7 +68,7 @@ export type AppState = {
   currentItemStrokeStyle: ExcalidrawElement["strokeStyle"];
   currentItemRoughness: number;
   currentItemOpacity: number;
-  currentItemFontFamily: FontFamily;
+  currentItemFontFamily: FONT_FAMILY;
```

Comment:
since this is storing a number and not name of font-family, should we rename this to `currentItemFontFamilyIndex` ?

Reply:
I'd keep this as is. The underlying data structure shouldn't affect the naming.